src/components/Parent/Parent.tsx

```diff
@@ -0,0 +1,17 @@
+import { PersonLink } from '../PersonLink';
+import { Person } from '../../types';
+
+type Props = {
+  parent: Person | null;
+  parentName: string | null;
+};
```

Comment:
just note:
you can do prop as not required, so null can be simplified
```suggestion
type Props = {
  parent?: Person;
  parentName?: string;
};
```